cli/src/wallet.rs

```diff
@@ -271,7 +271,7 @@ impl WalletSubCommands for App<'_, '_> {
                         .takes_value(true)
                         .validator(is_amount_or_all)
                         .required(true)
-                        .help("The amount to send, in SOL; accepts keyword ALL"),
+                        .help("The amount to send, in SOL, truncated to 9 decimals; accepts keyword ALL"),
```

Comment:
nit: If you want to make this change, it should be done in all places where we use the `is_amount`, `is_amount_or_all`, or `is_amount_or_all_or_available` validators.

Reply:
I'm ambivalent here; I'd only make it at all if we think someone out there is using scripts that rely on the old behavior, but I'm not familiar enough to know.

If that seems vanishingly unlikely, brevity is better.

Reply:
I'd prefer reverting it to be honest. We can shout out the change in the changelog though: https://github.com/anza-xyz/agave/blob/master/CHANGELOG.md -- what do you think?

Reply:
Sure, changelog is perfect, will wrap up shortly